src/commands/create-entur-tag.yml

```diff
@@ -16,6 +16,6 @@ steps:
           SHORT_BRANCH_NAME=<< parameters.branch_name >>
         fi
           # shellcheck disable=SC2021
-          TAG=$(echo "$SHORT_BRANCH_NAME"."$(date '+%Y%m%d')"-"${CIRCLE_SHA1:0:7}"-"${CIRCLE_WORKFLOW_ID:0:4}" | tr '[:upper:]/_' '[:lower:]--' | tr -d '[ÆØÅæøå]')
+          TAG=$(echo "$SHORT_BRANCH_NAME"."$(date +%Y-%m-%dT%H%M)"-"${CIRCLE_SHA1:0:7}"-"${CIRCLE_WORKFLOW_ID:0:4}" | tr '[:upper:]/_' '[:lower:]--' | tr -d '[ÆØÅæøå]')
```

Comment:
Syntes det burde legges på en `%z` for å vite hvilken tidssone man er i :)
Typ `date +%Y-%m-%dT%H%M%z`

Reply:
![image](https://github.com/entur/circleci-tools-orb/assets/5319700/ec07dc30-fe51-4b75-b3e1-1f654ee32693)